sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1743,6 +1743,17 @@ object functions {
 
   /**
    * Format strings in printf-style.
+   *
+   * @group string_funcs
+   * @since 1.5.0
+   */
+  @scala.annotation.varargs
+  def formatString(format: Column, arguments: Column*): Column = {
```

Comment:
i was thinking about a a version in which format is a string, but rest of the arguments are columns.